cumulusci/tasks/salesforce/ListCommunities.py

```diff
@@ -0,0 +1,42 @@
+from rst2ansi import rst2ansi
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class ListCommunities(BaseSalesforceApiTask):
+    api_version = "46.0"
+    task_docs = """
+    Lists Communities for the current org via the Connect API.
+    """
+    task_options = {}
+
+    def _init_options(self, kwargs):
+        super(ListCommunities, self)._init_options(kwargs)
```

Comment:
There's no reason to include this since you're not changing the behavior from the super class.